FactorioWebInterface/Program.cs

```diff
@@ -95,7 +97,68 @@ private static void SeedData(IHost host)
 
                 roleManager.CreateAsync(new IdentityRole(Constants.RootRole));
                 roleManager.CreateAsync(new IdentityRole(Constants.AdminRole));
+
+                _ = DefaultUserAsync(services);
+            }
+        }
+
+        private static async Task DefaultUserAsync(IServiceProvider services)
+        {
+            DefaultAdminAccountOption option = services.GetRequiredService<IOptions<DefaultAdminAccountOption>>().Value;
+
+            Console.WriteLine(option.Enabled.ToString());
```

Comment:
This could be a Log.Info.
Logging should write to the console, see line 34 above.

Reply:
I just forgot to remove it
```suggestion
```